lib/oppen.rb

```diff
@@ -103,17 +103,17 @@ def self.wadler(eager_print: true, upsize_stack: true)
   # @param value [String]
   #
   # @return [Oppen::Token::String] a new String token
-  def self.string(value)
-    Token::String.new(value)
+  def self.string(value, width = value.length)
```

Comment:
I would make it a named argument.

Is value guaranteed to be a string?  Don't you need to_s?

Reply:
I don't care much about the calling convention of others.  We should just be clear.  It is much easier to read with named argument.

But anyway this will be settled when Firas is back, proceed.